lib/Frontend/CompilerInvocation.cpp

```diff
@@ -34,6 +34,8 @@ using namespace llvm::opt;
 /// The path for Swift libraries in the OS on Darwin.
 #define DARWIN_OS_LIBRARY_PATH "/usr/lib/swift"
 
+static constexpr const char * const supportedLocaleCodes[] = {"en", "fr"};
```

Comment:
Should be `en` for now.

Reply:
@xedin to accommodate `llvm::join` I'll need to change `supportedLocaleCodes` to be of type `std::vector` i.e.
`static const std::vector<std::string> supportedLocaleCodes = {"en"};` 
Do you think it's okay to do that?

Reply:
I think you don't need to do that if you'd use `llvm::interleave`. It's possible to construct an `ArrayRef` form array e.g. http://llvm.org/doxygen/classllvm_1_1ArrayRef.html#a89d9c860c35ef271b626d13d1e356bd0

Reply:
Yes, I was just looking at some examples. Thank you :)

Reply:
Actually after thinking about this some more we might want to include all of the languages with their proper codes otherwise it would be impossible to do testing... My suggest to change this list to `en` only would break new tests :/

Reply:
Note that we should do that as a `.def` file as well to avoid polluting this file with a much of codes and that would make it easier to compute `NumLocales` just like we do for diagnostic messages.